encord/client.py

```diff
@@ -225,14 +230,15 @@ def initialise_with_config(config: ApiKeyConfig) -> Union[EncordClientProject, E
         """
         querier = Querier(config, resource_id=config.resource_id)
         key_type = querier.basic_getter(ApiKeyMeta)
+        api_client = ApiClient(config)
```

Comment:
nah this ain't right. with ApiKeys we can't call the new stuff so no point creating it.